docs/Reference/quorum.js/RawTransactionManager.md

```diff
@@ -1,213 +0,0 @@
----
```

Comment:
The web3js-quorum equivalent of this (including method descriptions) can be found in the web3js-quorum documentation.